src/main/java/com/github/dockerjava/core/command/CreateContainerCmdImpl.java

```diff
@@ -267,6 +267,12 @@ public Integer getCpuShares() {
         return hostConfig.getDnsSearch();
     }
 
+    @Override
+    @JsonIgnore
+    public List<String> getGroupAdd() {
```

Comment:
don't proxy host changes, keep only getter/setters in hostconfig

Reply:
Would that not make it awkward to use?
I saw several parts of hostconfig being proxied, is there a practice for which of them to expose?